src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -235,6 +236,12 @@ where automation.Name.Equals(calculateLibraryFuncName)
                
 
             stepUIPopup.IsOpen = true;
+
+            var nextButton = stepUIPopup.FindName(NextButton) as Button;
```

Comment:
I found this code which doing similar, https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs#L368, can you take a look?

Reply:
changed it to use the FindChild API on Guide class.

Reply:
Hi @reddyashish Seems we need a `null` similar to the code example above